velox/functions/prestosql/tests/ComparisonsTest.cpp

```diff
@@ -52,3 +52,33 @@ TEST_F(ComparisonsTest, betweenVarchar) {
   EXPECT_EQ(true, between("orange"));
   EXPECT_EQ(true, between("pear"));
 }
+
+TEST_F(ComparisonsTest, betweenDate) {
+  Date beforeJune2019;
```

Comment:
This test would be shorter and easier to read if parseTo() took a string and returned a Date(). 

````
  std::vector<std::tuple<Date, bool>> testData = {
      {parse("2019-05-01"), false},
      {parse("2019-06-01"), true},
      ...};
```